lms/templates/vert_module.html

```diff
@@ -17,3 +17,20 @@ <h2 class="hd hd-2 unit-title">${unit_title}</h2>
   %endif
 % endfor
 </div>
+
+<script>
+  $(function(){
+    var ext = '.csv';
+    $('a[href*="' + ext + '"]').each(function() {
+      var hrefVal = this.href;
+      if(hrefVal && hrefVal != '') {
+        var fileName = hrefVal.split('/').pop().split('.');
+        if (typeof fileName[0] !== "undefined") {
```

Comment:
combine ifs using and `&&`

Reply:
if (typeof fileName[0] !== "undefined") {
          if(fileName[0] != '') {
          
combining both ifs will produce error in case of undefined fileName[0] for the second condition, but I am eliminating ifs now, so not needed.